nuget/lib/dependabot/nuget/native_helpers.rb

```diff
@@ -103,14 +97,30 @@ def self.run_nuget_updater_tool(repo_root:, proj_path:, dependency:, is_transiti
           "--verbose"
         ].compact.join(" ")
 
+        [command, fingerprint]
+      end
+
+      sig do
+        params(
+          repo_root: String,
+          proj_path: String,
+          dependency: Dependency,
+          is_transitive: T::Boolean,
+          credentials: T::Array[T.untyped]
+        ).void
+      end
+      def self.run_nuget_updater_tool(repo_root:, proj_path:, dependency:, is_transitive:, credentials:)
+        (command, fingerprint) = get_nuget_updater_tool_command(repo_root: repo_root, proj_path: proj_path,
+                                                                dependency: dependency, is_transitive: is_transitive)
+
+        command = T.must(command)
```

Comment:
nit: you can directly put this as part of the method call below. But it's a stylistic choice, so feel free to ignore this.